monkey/monkey_island/cc/services/reporting/report.py

```diff
@@ -65,7 +65,7 @@ class ISSUES_DICT(Enum):
         VSFTPD = 13
         DRUPAL = 14
         ZEROLOGON = 15
-        ZEROLOGON_PASSWORD_RESTORED = 16
+        ZEROLOGON_PASSWORD_RESTORE_FAILED = 16
```

Comment:
I'm not sure I understand the purpose of this change. I think makes the code harder to follow. I think the code is cleaner if the logic of the telemetry and view are consistent.

Reply:
I've tried to document the purpose of this change in the commit message. This is an `ISSUE_DICT`. `ZEROLOGON_PASSWORD_RESTORED` is not an issue. `ZEROLOGON_PASSWORD_RESTORE_FAILED` is an issue and we display warnings if it's `true`, same as with other issues. In report service we create this enum to specify which issues happened and which issues did not so the front end only needs to check `if true render warning`

Reply:
OK, makes sense.